src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -36,23 +34,22 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_STUDENT_ID, PREFIX_PHONE, PREFIX_EMAIL,
-                        PREFIX_ADDRESS, PREFIX_TAG);
+                        PREFIX_TAG);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_STUDENT_ID, PREFIX_ADDRESS,
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_STUDENT_ID,
                 PREFIX_PHONE, PREFIX_EMAIL, PREFIX_STUDENT_ID) || !argMultimap.getPreamble().isEmpty()) {
```

Comment:
Hey Daryl, the student id prefix here is repeated, could you just amend that for me?

Reply:
Sure thing Glenn! I have amended to remove the repeat!